e2e_test/batch/duckdb/aggregate/aggregates/test_bit_xor.slt.part

```diff
@@ -0,0 +1,53 @@
+# Copied from https://github.com/duckdb/duckdb (MIT licensed).
+# Copyright 2018-2022 Stichting DuckDB Foundation
+
+# test on scalar values
+query II
+SELECT BIT_XOR(3), BIT_XOR(NULL::int)
```

Comment:
`::int` added by me, is it a known limitation?

Reply:
Yes. Because agg calls are not using the rules here yet:
https://github.com/risingwavelabs/risingwave/blob/6d2e75f8b14f1123c7be67fa5f5c0a2bbdf840bb/src/frontend/src/expr/type_inference/func.rs#L661